src/map/lua/luautils.cpp

```diff
@@ -2202,7 +2202,7 @@ namespace luautils
     {
         DSP_DEBUG_BREAK_IF(PSpell == nullptr);
 
-        PTarget->PAI->EventHandler.triggerListener("MAGIC_TAKE", PCaster, PTarget, PSpell);
+        PTarget->PAI->EventHandler.triggerListener("MAGIC_TAKE", PTarget, PCaster, PSpell);
```

Comment:
kind of odd that onMagicHit() always had the target before the caster. I guess because in those cases "mob" would have been the target and most mob functions have mob 1st in the existing param orders of other functions.